docs/getting-started.rst

```diff
@@ -4,101 +4,244 @@
 
-    * text fields in Pfam entries, e.g. family descriptions
-    * UniProt sequence entry description and species fields
-    * HEADER and TITLE fields from PDB entries
-    * Gene Ontology IDs and terms
-    * InterPro entry abstracts
+* **Curation**
+This tab is divided into two subsections:
+
+1. In the first section, you can see details about Pfam curators and Sequence ontology.
+
+2. The second section displays the HMM building command used to generate the HMM profile defining the signature.
 
-Each Pfam-A entry is listed only once in the results table, although it might have been found in more than one area of the database.
 
 Searching a protein sequence against Pfam
 =========================================
 
-Searching a protein sequence against the Pfam library of HMMs will enable you to find out the domain architecture of the protein. If your protein is present in the version of UniProt, NCBI Genpept or the metagenomic sequence set that we used to make the current release of Pfam, we have already calculated its domain architecture. You can access this by entering the sequence accession or ID in the 'view a sequence' box on the Pfam homepage.
+Searching a protein sequence against the Pfam library of HMMs will enable you to find out the domain architecture of the protein, and thus what its potential function might be. If your protein is present in the version of UniProt used to make the current release of InterPro, we have already calculated its domain architecture. You can access this by entering the Uniprot sequence identifier in any of the Search boxes mentioned above.
 
-If your sequence is not in the Pfam database, you could perform a single-sequence or a batch search by clicking on the 'Search' link at the top of the Pfam page.
+Web services
+------------
 
-Single protein search
----------------------
+If your sequence is not in the InterPro database, you could perform a single-sequence or a batch search against the Pfam database on the InterPro website. This search uses the web based InterProScan tool, which allows you to scan up 100 sequences at a time with a maximum length of 40,000 amino acids. To run a any online search you can follow this steps:
 
-If your protein is not recognised by Pfam, you will need to paste the protein sequence into the search page. We will search your sequence against our HMMs and instantly display the matches for you.
+1. Click the **Search + By Sequence** in the InterPro website menu. This opens the **Search InterPro** page. 
 
-Batch search
-------------
+.. figure:: images/searchbyseq.png
+    :alt: Selecting **Search + By Sequence** in the InterPro website menu
+    :width: 700
+    :align: left
+
+    Selecting **Search + By Sequence** in the InterPro website menu.
+
+2. Provide the FASTA formatted sequence(s) (protein or DNA) of interest by pasting them into the test dialogue box or by importing them from a file
```

Comment:
we only support protein sequences atm
typo: text
remove `dialogue`

Reply:
I think I asked about this when doing the tutorial but not sure... DNA sequences were only accepted when running InterProScan locally? or not there either?
I already changed the text here, this is about web services so it was wrong, but wondering if the corresponding needs update in the tutorial..

Reply:
The DNA sequences are only supported in the local install of InterProScan indeed.